etl/steps/data/garden/education/2023-08-14/oecd_education.countries.json

```diff
@@ -0,0 +1,37 @@
+{
+  "ARG": "Argentina",
+  "AUS": "Australia",
+  "BRA": "Brazil",
+  "CAN": "Canada",
+  "CHN": "China",
+  "DEU": "Germany",
+  "EGY": "Egypt",
+  "ESP": "Spain",
+  "FRA": "France",
+  "GBR": "United Kingdom",
+  "IDN": "Indonesia",
+  "IND": "India",
+  "ITA": "Italy",
+  "JPN": "Japan",
+  "KEN": "Kenya",
+  "MEX": "Mexico",
+  "NGA": "Nigeria",
+  "NLD": "Netherlands",
+  "POL": "Poland",
+  "RUS": "Russia",
+  "SWE": "Sweden",
+  "THA": "Thailand",
+  "TUR": "Turkey",
+  "USA": "United States",
+  "World": "World",
+  "ZAF": "South Africa",
+  "East Asia (EA)": "East Asia",
+  "Eastern Europe    (EE)": "Eastern Europe",
```

Comment:
I would add `(OECD)` to the more the regions to indicate that they are based on the OECD's definitions of the regions rather than ours.